src/less/main.less

```diff
@@ -2496,3 +2497,71 @@ div.shake-alert {
     height: 0;
   }
 }
+
+// Billing
+.billing {
+  display: grid;
+  grid-template-columns: (2 * @defaultLeftBarWidth) 1fr;
+  > div {
+    padding: (2 * @defaultGutterWidth) (3 * @defaultGutterWidth);
+    margin-top: (-2 * @defaultGutterWidth);
+  }
+  .explanatory-text {
+    color: @verylightgrey;
+  }
+  .billing-subtitle {
+    margin-bottom: (0.5 * @defaultGutterWidth);
+  }
+  .usage-report {
+    background-color: @lightestgrey;
+    position: relative;
+    .overlay {
+      position: absolute;
+      margin: (-2 * @defaultGutterWidth) (-3 * @defaultGutterWidth);
+      backdrop-filter: blur(3px);
+      z-index: 100;
+      width: 100%;
+      height: 100%;
```

Comment:
Could you add
```
padding: 0 30px;
background-color: rgba(247, 247, 247, 0.5)
```
to .overlay? Just to help a bit with readability

Reply:
done!